.github/workflows/release-build.yml

```diff
@@ -25,6 +25,11 @@ jobs:
         run: echo "VERSION_NO_DOTS=$(echo $VERSION | sed s/\\.//g)" >> $GITHUB_ENV
       - name: Build gem
         run: gem build aws-xray-sdk.gemspec
+      - name: Set API key credentials
+        run: |
+          echo ":rubygems_api_key: $RUBYGEMS_API_KEY" > ~/.gem/credentials
```

Comment:
Should we shred the `credentials` after we are done with the release to make sure it is not lying around on the runner instance?

Reply:
Yeah good idea